src/components/Molecules/Card/Card.js

```diff
@@ -1,27 +1,64 @@
 import React from 'react';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import PropTypes from 'prop-types';
 import Picture from '../../Atoms/Picture/Picture';
 import spacing from '../../../theme/shared/spacing';
 
 const Container = styled.div`
-  display: flex;
   position: relative;
-  flex-direction: column;
+  display: flex;
```

Comment:
Just as part of housekeeping since now that the Card styles are much bigger, could we move them into their own `Card.styles.js` file plz?